website/docs/r/vra_block_device_snapshots.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: "vra"
+page_title: "VMware vRealize Automation: vra_block_device"
+description: |-
+  Provides a VMware vRA vra_block_device resource.
+---
+
+# Resource: vra_block_device_snapshots
+## Example Usages
+
+This is an example of how to create a block device snapshot resource.
+
+```hcl
+resource "vra_block_device_snapshot" "snapshot1" {
+  block_device_id = vra_block_device.disk1.id
```

Comment:
Change this to `block_device_id = var.block_device_id`.

Also, change the file name to `vra_block_device_snapshot.html.markdown`